context/app/static/js/components/detailPage/ProcessedData/HelperPanel/HelperPanel.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { PropsWithChildren } from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { formatDate } from 'date-fns/format';
+import Divider from '@mui/material/Divider';
+import { useIsDesktop } from 'js/hooks/media-queries';
+import SchemaRounded from '@mui/icons-material/SchemaRounded';
+import { WorkspacesIcon } from 'js/shared-styles/icons';
+import CloudDownloadRounded from '@mui/icons-material/CloudDownloadRounded';
+import { useAppContext } from 'js/components/Contexts';
+import { formatSectionHash } from 'js/shared-styles/sections/TableOfContents/utils';
+import { useAnimatedSidebarPosition } from 'js/shared-styles/sections/TableOfContents/hooks';
+import { animated } from '@react-spring/web';
+import { ListItemIcon, Menu, MenuItem } from '@mui/material';
+import AddRounded from '@mui/icons-material/AddRounded';
+import NewWorkspaceDialog from 'js/components/workspaces/NewWorkspaceDialog';
+import { useCreateWorkspaceForm } from 'js/components/workspaces/NewWorkspaceDialog/useCreateWorkspaceForm';
+import { useOpenDialog } from 'js/components/workspaces/WorkspacesDropdownMenu/WorkspacesDropdownMenu';
+import SelectableTableProvider from 'js/shared-styles/tables/SelectableTableProvider/SelectableTableProvider';
+import AddDatasetsFromSearchDialog from 'js/components/workspaces/AddDatasetsFromSearchDialog';
+import { HelperPanelPortal } from '../../DetailLayout/DetailLayout';
+import useProcessedDataStore from '../store';
+import StatusIcon from '../../StatusIcon';
+import { getDateLabelAndValue } from '../../utils';
+import { HelperPanelButton } from './styles';
+import { useTrackEntityPageEvent } from '../../useTrackEntityPageEvent';
+
+function useCurrentDataset() {
+  return useProcessedDataStore((state) => state.currentDataset);
+}
+
+function HelperPanelHeader() {
+  const currentDataset = useCurrentDataset();
+  return (
+    <Typography variant="subtitle2" display="flex" alignItems="center" gap={0.5} whiteSpace="nowrap">
+      <SchemaRounded fontSize="small" />
+      <a href={`#${formatSectionHash(`section-${currentDataset?.hubmap_id}`)}`}>{currentDataset?.hubmap_id}</a>
+    </Typography>
+  );
+}
+
+function HelperPanelStatus() {
+  const currentDataset = useCurrentDataset();
+  if (!currentDataset) {
+    return null;
+  }
+  return (
+    <Stack direction="row" alignItems="center">
+      <StatusIcon status={currentDataset.status} />
+      <Typography variant="body2">{currentDataset.status}</Typography>
+    </Stack>
+  );
+}
+
+interface HelperPanelBodyItemProps extends PropsWithChildren {
+  label: string;
+  noWrap?: boolean;
+}
+
+const noWrapStyles = {
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  display: '-webkit-box',
+  WebkitLineClamp: 3,
+  WebkitBoxOrient: 'vertical',
+};
```

Comment:
Should we reuse the `LineClamp` component?

Reply:
Genuinely forgot about that already existing despite being the one to add it 😆